crates/chain/src/tx_data_traits.rs

```diff
@@ -5,21 +5,22 @@ use alloc::vec::Vec;
 
 /// Trait that "anchors" blockchain data to a specific block of height and hash.
 ///
-/// [`Anchor`] implementations must be [`Ord`] by the anchor block's [`BlockId`] first.
-///
-/// I.e. If transaction A is anchored in block B, then if block B is in the best chain, we can
+/// If transaction A is anchored in block B, then if block B is in the best chain, we can
```

Comment:
```suggestion
/// If transaction A is anchored in block B, and if block B is in the best chain, then we can
```

Reply:
Nit: I would also remove the `if`.

```suggestion
/// If transaction A is anchored in block B, and block B is in the best chain, we can
```